animations/README.md

```diff
@@ -23,7 +23,7 @@ Building blocks and patterns
 6. **CustomTweenDemo** Demonstrates how to extend `Tween`.
 7. **TweenSequenceDemo** Demonstrates how to use `TweenSequence` to build a
 button that changes between different colors.
-8. **AnimatedList** Demonstrates how to use `AnimatedList`
+8. **HeroAnimationDemo** Demonstrates how to use `Hero` animation.
```

Comment:
This would likely fit better into the misc. list below. The basics are intended to a be a specifically ordered tour of animations concepts.